phpseclib/Crypt/RSA.php

```diff
@@ -2154,9 +2215,10 @@ function encrypt($plaintext)
      * @see self::encrypt()
      * @access public
      * @param string $plaintext
+     * @param int|bool $padding
```

Comment:
Should this be `@param int $padding`, like in the case of `encrypt()` and `sign()`?